jenkins/post-build-status-update.jenkinsfile

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env groovy
+
+import groovy.json.JsonOutput
+
+pipeline {
+    agent { label 'jenkins-workspace' }
+
+    parameters {
+        string(defaultValue: '', description: 'test/aix, linter, etc.', name: 'IDENTIFIER')
```

Comment:
Not a change request, just noting for future reference that the current list appears to be (not sure where else it's written down):

```
linter
test/aix
test/arm-fanned
test/freebsd
test/linux
test/linux-fips
test/linux-one
test/osx
test/ppc-linux
test/smartos
test/windows-fanned
```

Reply:
It's also available [here](https://github.com/maclover7/post-ci-status/blob/master/run.rb#L5-L17).